runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2018, Tanner <https://github.com/Reasel>
  * Copyright (c) 2016-2017, Abel Briggs
  * Copyright (c) 2017, Kronos <https://github.com/KronosDesign>
```

Comment:
Move your name to the bottom of the attribution list, they're chronological.

Reply:
Done.